app/views/home/_sponsors.ejs

```diff
@@ -1,77 +1,73 @@
-<section id="sponsors" class='o-section o-box'>
-  <div class='o-wrapper u-mt++'>
-    <h1 class='o-section__title'>Sponsors</h1>
+<section id="sponsors" class='o-section u-mv++ u-pb++'>
+  <div class='o-wrapper u-text-align--center'>
+    <h3 class='u-color--primary u-font-weight--300 u-mb++'>Sponsors</h1>
+    <h4>Gold</h4>
 
-    <div class='c-sponsors u-mv++'>
-      <h2 class='c-sponsors__type'>Gold</h2>
-      <ul class='o-layout'>
-        <li class='o-layout__item u-1-of-1 u-1-of-3-start u-1-of-5-small'>
-          <a href="http://genexus.com" target="_blank">
-            <img class="sponsor-gold u-mv+" src="logos/genexus.png" alt="Genexus"/>
-          </a>
-        </li>
-        <li class='o-layout__item u-1-of-1 u-1-of-3-start u-1-of-5-small'>
-          <a href="http://codigodelsur.com" target="_blank">
-            <img class="sponsor-gold u-mv+" src="logos/codigo_del_sur.png" alt="Código del Sur"/>
-          </a>
-        </li>
-        <li class='o-layout__item u-1-of-1 u-1-of-3-start u-1-of-5-small'>
-          <a href="https://moove-it.com" target="_blank">
-            <img class="sponsor-gold u-mv+" src="logos/moove-it.png" alt="Moove-it"/>
-          </a>
-        </li>
-        <li class='o-layout__item u-1-of-1 u-1-of-3-start u-1-of-5-small'>
-          <a href="http://mercadolibre.com" target="_blank">
-            <img class="sponsor-gold u-mv+" src="logos/mercado_libre.png" alt="Mercado Libre"/>
-          </a>
-        </li>
-        <li class='o-layout__item u-1-of-1 u-1-of-3-start u-1-of-5-small'>
-          <a href="#url-marca" target="_blank">
-            <img class="sponsor-gold u-mv+" alt="Sponsor GOLD"/>
-          </a>
-        </li>
-      </ul>
-    </div>
+    <ul class='o-pack o-pack--middle u-m0 u-p0'>
```

Comment:
Se perdió la adaptación responsive de los items de las listas de sponsors. Veo que las clases u-1-of-1, u-1-of-3-start, u-1-of-5-small siguen estando con la intención de hacer esto pero por alguna razón no terminan siendo aplicadas.